.github/workflows/release.yaml

```diff
@@ -25,7 +26,7 @@ jobs:
       - build
     uses: canonical/data-platform-workflows/.github/workflows/release_snap.yaml@v30.1.3
     with:
-      channel: 14/edge
+      channel: ${{ github.ref_name }}
```

Comment:
Alternatively we can have a permanent diff here.

Reply:
I'm fine with either